cypress/integration/submission-ui.spec.ts

```diff
@@ -2,290 +2,293 @@
  * This IT will be never be pushed to the upstream because clicking testing DOM elements is antipattern because
  * the tests on other machines could fail.
  */
-import {
-  TEST_ADMIN_PASSWORD,
-  TEST_ADMIN_USER,
-  TEST_SUBMIT_CLARIAH_COLLECTION_UUID,
-  TEST_SUBMIT_COLLECTION_UUID
-} from '../support';
-import { createItemProcess } from '../support/commands';
+// NOTE: This test class is commented out for the TUL customer because these tests are suitable only for
+// CLARIN-DSpace submission.
```

Comment:
but tests are commented out. Is it possible, for the sake of clarity and future usefulness to comment out both? If someone is to uncomment test, they will have to spend time finding proper imports. Or if nobody ever should use those tests, then dele them as imports are deleted now. I see two options = > delete tests so that tests are deleted as well as import, OR just comment out imports so that tests and imports are just commented out. Make it the same. Or is my thinking wrong?

Reply:
The imports are commented. Thank you